### PR TITLE
Remove Enterprise Only flag

### PR DESCRIPTION
**Pull request by petems**
* Available in OSS Vault from 0.10+

### CODE REVIEW

File: website/source/docs/configuration/index.html.md [hunk 1]
```diff
@@ -121,7 +121,7 @@ to specify where the configuration is.
   allows the decryption/encryption of raw data into and out of the security
   barrier. This is a highly privileged endpoint.
 
-- `ui` `(bool: false, Enterprise-only)` – Enables the built-in web UI, which is
```
Comment (petems) [inline website/source/docs/configuration/index.html.md]: Do we want to highlight this as only avaliable in 0.10+?

Comment (jefferai) [reply website/source/docs/configuration/index.html.md]: No -- our docs don't really indicate versions (except in a few places, which they aren't consistent about). A higher level versioned-docs project seems to be slowly gathering steam, which will address this.

Comment (jefferai) [reply website/source/docs/configuration/index.html.md]: That said, if you want to note it, I'm fine with it.

Comment (petems) [reply website/source/docs/configuration/index.html.md]: Cool, I think a brief note would be good, with the "real" fix coming with the versioned docs work in the future

**Pull request merged**